ecies.c

```diff
@@ -19,14 +19,6 @@ const char key[] = \
 "nzwv/wzEUsNPrLSUfMq1dYvt3hk=\n" \
 "-----END PRIVATE KEY-----\n";
 
-void RAND_init(void) {
```

Comment:
Especially for one who not too enthusiastic and too picky simultaneously.
There are at least 3 reasons for that:
1. `/dev/random` does not exist on windows.
2. Code should not call `/dev/random`. The right way is to use `/dev/urandom` instead. Because of potential program hang due to lack of user input.
3. OpenSSL calls `RAND_seed` automatically in a platform dependent manner by itself  when you call `RAND_bytes`. But as it said in  OpenSSL docs for production use we can't rely on that and should pay great attention to seeding random numbers. This will require far more greater efforts than just a build fix.

Anyway this is an "example not for production" and It's up to you accept this pull request or not.